updater/lib/dependabot/updater/operations/update_all_versions.rb

```diff
@@ -99,6 +100,12 @@ def check_and_create_pull_request(dependency)
             requirements_to_unlock: requirements_to_unlock
           )
 
+          if updated_deps.empty?
+            return Dependabot.logger.info(
+              "No update possible for #{dependency.name} #{dependency.version}"
+            )
+          end
+
```

Comment:
Seems like another tweak that's probably a good change for us, but probably unrelated to NuGet, so best to extract it to a separate PR.